api/soap/mantisconnect.wsdl

```diff
@@ -288,6 +332,20 @@
    <xsd:element name="require_closed" type="xsd:boolean" minOccurs="0"/>
   </xsd:all>
  </xsd:complexType>
+ <xsd:complexType name="FilterCustomField">
+  <xsd:all>
+   <xsd:element name="name" type="xsd:string" minOccurs="0"/>
```

Comment:
We typically use ObjectRef for cases where a user can supply an id or a name to reference an entity.